client/src/ui/MouseEvent.res

```diff
@@ -0,0 +1,31 @@
+// TODO: can this be replaced with a type from a library?
+@ppx.deriving(show({with_path: false}))
+type rec t = {
+  mePos: VPos.t,
+  button: int,
```

Comment:
What's `button` here - some encoding of the buttons that are clicked/held?

Reply:
Probably? Again just moving this around